model/aspnetcore/metrics.yaml

```diff
@@ -135,3 +141,298 @@ groups:
     attributes:
       - ref: aspnetcore.rate_limiting.result
         requirement_level: required
+
+  # identity
+  - id: metric.aspnetcore.identity.user.create
+    type: metric
+    metric_name: aspnetcore.identity.user.create
+    annotations:
+      code_generation:
+        metric_value_type: int
+    stability: stable
+    brief: The number of users created.
+    instrument: counter
+    unit: "{count}"
```

Comment:
I use `{count}` in these metrics as the unit.

Would a more descriptive unit be better? For example, `{user}` for this method

Reply:
yep, more precise unit would be better (where it makes sense)